backend/flask_api.py

```diff
@@ -82,7 +79,7 @@ class User(db.Document, UserMixin):
 
 class EmailConverter(BaseConverter):
     regex = r"([a-z0-9!#$%&'*+\/=?^_`{|}~-]+(?:\." +\
-            r"[a-z0-9!#$%&'*+\/=?^_`""{|}~-]+)" +\
+            r"[a-z0-9!#$%&'*+\/=?^_`"r"{|}~-]+)" +\
```

Comment:
make it a raw string literal, it isn't necessary there but goes in style with the other r's I added (which I had to, \ and other letters have to be escaped otherwise which they weren't)